const.go

```diff
@@ -13,4 +13,8 @@ const (
 
 	// MaxSatoshi is the maximum transaction amount allowed in satoshi.
 	MaxSatoshi = 21e6 * SatoshiPerBitcoin
+
+	// Satoshi is the base bitcoin monetary unit.
+	// A single amount is equal to 1e-8 of a bitcoin.
```

Comment:
The workaround isn't needed in the first place since you can just type convert integer satoshi amounts to Amount.

```Go
var satoshis int64 = 1.23e8
var amount = btcutil.Amount(satoshis)
fmt.Println(amount == 1.23e8) // true
```